skydb/skydb.go

```diff
@@ -162,3 +165,16 @@ func uploadData(c *client.Client, content []byte) (string, error) {
 	}
 	return skylink, nil
 }
+
+// waitUntilSkydReady checks the /daemon/ready endpoint and waits until skyd is
+// fully ready
+func waitUntilSkydReady(c *client.Client) {
+	for {
```

Comment:
should this give up at some point? e.g. after 5 minutes or so?

Reply:
I don't know. If the portal node has been down for a bit it might need time to sync up. If we add a hard limit I would set it to something high like 15 minutes. But before skyd is ready we can't do anything here.

Reply:
F/U: https://github.com/SkynetLabs/certmagic-storage-skydb/pull/4